dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -90,14 +104,19 @@
                                 <VisualState x:Name="LastPageButtonEnabled"/>
                                 <VisualState x:Name="LastPageButtonDisabled">
                                     <VisualState.Setters>
-                                        <Setter Target="LastPageButton.IsEnabled" Value="false"/>
+                                        <Setter Target="LastPageButton.IsEnabled" Value="False"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="LastPageButtonHidden">
+                                    <VisualState.Setters>
+                                        <Setter Target="LastPageButton.Opacity" Value="0"/>
+                                        <Setter Target="LastPageButton.IsEnabled" Value="False"/>
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
                             <VisualStateGroup x:Name="PagerDisplayModeStates">
                                 <VisualState x:Name="NumberBoxVisible">
                                     <VisualState.Setters>
-                                        <Setter Target="NumberBoxDisplay.IsEnabled" Value="True"/>
```

Comment:
>IsEnabled [](start = 73, length = 9)

why can this be deleted?

Reply:
Previously, the NumberBox was disabled and enable everytime we switch modes. However when we collapse the control, we don't need to disable it. Since we don't disable it anymore when hiding, we don't have to reenable it when showing it.